blocks/bricks/interfaces.py

```diff
@@ -196,8 +207,10 @@ def _initialize(self):
         # Use self.parameters[] references in case W and b are overridden
         # to return non-shared-variables.
         if self.use_bias:
-            self.biases_init.initialize(self.parameters[1], self.rng)
-        self.weights_init.initialize(self.parameters[0], self.rng)
+            self.initialization_schemes['biases_init'].initialize(
```

Comment:
Same question about what the keys here ought to be. Also the minor nit that the role names are "BIAS" and "WEIGHT" but for the current init args we use the plural.